.promu.yml

```diff
@@ -9,11 +9,11 @@ build:
           path: ./cmd/promtool
     flags: -tags netgo
     ldflags: |
-        -X {{repoPath}}/version.Version={{.Version}}
-        -X {{repoPath}}/version.Revision={{.Revision}}
-        -X {{repoPath}}/version.Branch={{.Branch}}
-        -X {{repoPath}}/version.BuildUser={{user}}@{{host}}
-        -X {{repoPath}}/version.BuildDate={{date "20060102-15:04:05"}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
```

Comment:
Which will add duplicated lines of code in each repository.
Indeed those flags looks nasty, but there no other choise for a vendored lib.
Moreover, this is a one time definition in the `.promu.yml` file, looks ugly but it works.